src/status_im/translations/en.cljs

```diff
@@ -157,10 +157,12 @@
    :discover                              "Discover"
    :none                                  "None"
    :search-tags                           "Type your search tags here"
-   :popular-tags                          "Popular tags"
-   :recent                                "Recent"
+   :popular-tags                          "Popular hashtags"
```

Comment:
What's the reason for switching to the more verbose hashtags? Is tags not clear enough?

Reply:
No idea, just typed what was in the screen designs at the time. @denis-sharypin any particular reson?

Reply:
Ok let's leave it then for this PR, probably a reason, was mostly curious if this was a deliberate decision or not